conf/broker.conf

```diff
@@ -800,6 +800,10 @@ managedLedgerDefaultWriteQuorum=2
 # Number of guaranteed copies (acks to wait before write is complete)
 managedLedgerDefaultAckQuorum=2
 
+# with OpportunisticStriping=true the ensembleSize is adapted automatically to writeQuorum
+# in case of lack of enough bookies
+bookkeeperOpportunisticStriping=false
```

Comment:
I think we should provide a more generic method to load and customize bookkeeper settings instead of adding them one by one.

Reply:
I am open to switch to a new mode of configuring the internal BK client.

If I understand correctly currently in Pulsar community we prefer to allow only a limited and known set of BK configuration options in order to have total control over which feature of BK can be safely used and that we are officially supporting in Pulsar 

If we let any configuration option to be activated users could fall into bad conditions that are not supposed to happen in Pulsar.

Can you confirm that a more generic approach is what we want?

Reply:
I think we should provide a generic approach instead of patching individual configurations.